Access Engineering.js

```diff
@@ -0,0 +1,334 @@
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.accessengineeringlibrary.com/content/case-study/CS0007_Modeling_the_2020_Novel_Coronavirus_Pandemic",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Modeling the 2020 Novel Coronavirus Pandemic",
+				"creators": [
+					{
+						"firstName": "Lori",
+						"lastName": "Herz",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Ryan",
+						"lastName": "Zurakowski",
+						"creatorType": "author"
+					}
+				],
+				"date": "2020-12-02",
+				"abstractNote": "Understanding how mathematical models can be used to predict and analyze the spread of infectious diseases is a topic of interest, especially in light of the COVID-19 pandemic. This case study provides an overview of epidemiological models, as well as an in-depth study of how to use the models and perform a goodness-of-fit analysis.",
+				"language": "en",
+				"libraryCatalog": "www.accessengineeringlibrary.com",
+				"url": "https://www.accessengineeringlibrary.com/content/case-study/CS0007_Modeling_the_2020_Novel_Coronavirus_Pandemic",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.accessengineeringlibrary.com/content/tutorial/T0002_Open_Channel_Flow_Calculations_with_the_Manning_Equation",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Open Channel Flow Calculations with the Manning Equation using Excel Spreadsheets",
+				"creators": [
+					{
+						"firstName": "P. E.",
+						"lastName": "Harlan H. Bengtson",
+						"creatorType": "author"
+					}
+				],
+				"date": "2014-02-01",
+				"abstractNote": "<p>This tutorial teaches the Manning equation and its use for uniform open channel flow calculations, including the hydraulic radius, Manning roughness coefficient, and normal depth. There are example problems and illustrations show how to use spreadsheets for the calculations.</p>",
+				"language": "en",
+				"libraryCatalog": "www.accessengineeringlibrary.com",
+				"url": "https://www.accessengineeringlibrary.com/content/tutorial/T0002_Open_Channel_Flow_Calculations_with_the_Manning_Equation",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.accessengineeringlibrary.com/content/video/V4005352521001",
+		"items": [
+			{
+				"itemType": "videoRecording",
+				"title": "123D Design: Cut Text Through a Plane",
+				"creators": [
+					{
+						"firstName": "Professor",
+						"lastName": "Lydia Cline",
+						"creatorType": "author"
+					}
+				],
+				"date": "2014",
+				"abstractNote": "<p>This video shows how to cut text through a plane with Combine/Subtract.</p>",
```

Comment:
Run `abstractNote` through `ZU.cleanTags()` to remove HTML tags.

Reply:
Fixed.